src/nodes/QskBoxRendererEllipse.cpp

```diff
@@ -1124,7 +1270,7 @@ void QskBoxRenderer::renderRectellipseBorder(
     const int stepCount = metrics.corner[ 0 ].stepCount;
     const int lineCount = 4 * ( stepCount + 1 ) + 1;
 
-    const auto line = allocateLines< Line >( geometry, lineCount );
+    const auto line = allocateLines< ColoredLine >( geometry, lineCount );
```

Comment:
Changing Line to ColoredLine looks wrong to me.

When having a QSGFlatColorMaterial there is only one color that is set with QSGFlatColorMaterial::setColor ( see QsBoxNode::setBoxData) and the corresponding geometry is using vertexes without extra color information ( running into a different shader then ): see QskBoxNode::setMonochrome.

Reply:
Yeah, this turned out to be just an oversight I made when I started working on this.